i18n/translations/en.json

```diff
@@ -1,4 +1,8 @@
 {
+  "account.changeName": "Change your name",
+  "account.delete": "Delete your account",
+  "account.delete.confirm": "Do you really want to delete your account? You will be logged out immediately. Your account cannot be recovered.",
+
```

Comment:
When working on this, it became clear how weird the translations are when you work on your own account. Of course we could keep it, but I wonder if it's making this PR that much simpler compared to the enhanced UX when you read: "Delete your account" instead of "Delete this user"

Reply:
Ok, fine with that :)